README.md

```diff
@@ -1,12 +1,44 @@
 # bulkfhirloader
 Loads FHIR bundles directly into Mongo database backing a FHIR server
 
-The commandline arguements are:
+## About
+SyntheticMass consists of two databases.  A Mongo database which stores the detailed patient information and a Postgres database that is used for analytics.  The bulkloader is designed to append data to the Mongo database and truncated and replace the data in Postgres.  This allows SyntheticMass to be loaded in increments.  What the bulkloader **can not** do is an update of the data in Mongo.  
+
+
+## Running the bulkloader
+The bulkloader is executed from the commandline.  It expects the following four arguments:
   - Path to the FHIR files
-    - Assumes that these files are JSON
-  - Name of the Mongo DD server
+    - Assumes that these files are JSON and are one the same machine that is running the bulkloader
+  - Name of the Mongo database server
   - Name of the databse
   - Postgres connection string
     - i.e. user:pwd@somehost/dname
 
-The bulkloader assumes that its target databases are empty.  See the **clear_mongo_collections.js** and **clear_postgres_tables.sql** scripts.
+Given that a load could take several hours it is best to run the bulkloader as a background process with *nohup*.  Nohup is a linux/unix command that allows a process to keep running even if the session that spawned that process ends.
+
+A sample commandline would look like:
+    nohup ./bulkloader /data/fhir localhost:27017 fhir pguser:pgpwd@localhost/fhir &
+
+    - This assumes that both the Mongo and Postgres databases are called fhir and are on the localhost
```

Comment:
For some reason GitHub's markdown parser is interpreting these bullet points as a code block, probably because of the indentation. List the sample command in triple quotes (```) to display it as a code block:

```
nohup ./bulkloader /data/fhir localhost:27017 fhir pguser:pgpwd@localhost/fhir &
```

Then list your bullet points using asterisks (*), without any indentation:

* point 1
* point 2
* point 3

See this guide for markdown tips: [Markdown Cheat Sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)